training/ilab-wrapper/ilab

```diff
@@ -8,21 +8,54 @@ export ENTRYPOINT="/opt/python3.11/venv/bin/ilab"
 export PARAMS=("$@")
 
 for dir in "$HOME/.cache" "$HOME/.config" "$HOME/.local"; do
```

Comment:
Should probably check if $HOME is set. Rather then creating /.cache and /.config

Reply:
Not sure we even still need this if statement. Will address in a separate PR